### PR TITLE
Remove "done!" notification

### PR DESCRIPTION
**Pull request by D7ry**
By configuring `notify_done` variable in opts, users can disable the "done!" notification that some may find a bit distracting.

### CODE REVIEW

**Comment by deathbeam** [comment]
I think way better way would be to remove the notify completely from where its being called now and just set default callback (thats currently by default nil) to be function that calls `vim.notify`.

**Review by deathbeam — CHANGES_REQUESTED** [review]
As i said in my response, should just be callback imo

**Comment by D7ry** [comment]
> As i said in my response, should just be callback imo

I believe just not setting the default callback is better; removed the notify completely!

**Comment by deathbeam** [comment]
Yea i agree, much cleaner solution

**Review by deathbeam — APPROVED** [review]

File: lua/CopilotChat/config.lua [hunk 1]
```diff
@@ -76,6 +76,7 @@ local select = require('CopilotChat.select')
 ---@field prompts table<string, CopilotChat.config.prompt|string>?
 ---@field window CopilotChat.config.window?
 ---@field mappings CopilotChat.config.mappings?
+
```
Comment (deathbeam) [inline lua/CopilotChat/config.lua:79]: Same here

File: lua/CopilotChat/chat.lua [hunk 2]
```diff
@@ -30,7 +30,7 @@ function CopilotChatFoldExpr(lnum, separator)
   return '='
 end
 
-local Chat = class(function(self, mark_ns, help, on_buf_create)
+local Chat = class(function(self, mark_ns, help, on_buf_create, config)
```
Comment (deathbeam) [inline lua/CopilotChat/chat.lua:33]: Im not sure why are you passing config here

Comment (D7ry) [reply lua/CopilotChat/chat.lua]: updated. the config was required previous for getting the option but no longer

**Review by deathbeam — CHANGES_REQUESTED** [review]
see comments

File: README.md [hunk 3]
```diff
@@ -203,6 +203,7 @@ Also see [here](/lua/CopilotChat/config.lua):
   error_header = '**Error** ', -- Header to use for errors
   separator = '---', -- Separator to use in chat
 
+
```
Comment (jellydn) [inline README.md:206]:
```suggestion
```

File: lua/CopilotChat/config.lua (hunk 1 above)
Comment (jellydn) [inline lua/CopilotChat/config.lua:79]:
```suggestion
```

**Comment by jellydn** [comment]
@all-contributors add @D7ry for code.

**Pull request merged**